chef/cookbooks/nova_dashboard/attributes/default.rb

```diff
@@ -23,6 +23,15 @@
 default[:nova_dashboard][:site_branding_link] = ""
 default[:nova_dashboard][:help_url] = "http://docs.openstack.org/"
 
+default[:nova_dashboard][:policy_file_path] = nil
```

Comment:
Probably best to use `""` instead of `nil` by default, because `nil.empty?` doesn't work.

(although it won't impact us that much as the default in the data bag is an empty string)